modules/ROOT/pages/cloudhub-manage-props.adoc

```diff
@@ -65,6 +65,13 @@ To create set an environment variable using an application property:
 .The arrow shows the variable on the *Properties* tab of the application *Settings* page.
 image::rtm-env-variables.png[*Properties* tab on the *Settings* page]
 
++
+[NOTE]
+In the *Text* view, use the backslash (`\`) character as an escape character.
+For example, to use `:` in key or value, use `\:` in the text.
+To use `\` in a key or value, use `\\` in the text.
+No escape character is required in list format.
```

Comment:
in list format -> in *List* view? feel it would be better to keep it consistent with *Text* view